### PR TITLE
Some beautification ;-)

### PR DESCRIPTION
**Pull request by mathieucarbou**

### CODE REVIEW

File: voltron-proxy/voltron-proxy-client/src/main/java/org/terracotta/voltron/proxy/client/VoltronProxyInvocationHandler.java [hunk 1]
```diff
@@ -74,12 +74,11 @@
         listeners.put(aClass, new CopyOnWriteArrayList<MessageListener>());
       }
 
-      entityClientEndpoint.setDelegate(new EndpointDelegate() {
+      entityClientEndpoint.setDelegate(new EndpointDelegate<ProxyEntityResponse>() {
```
Comment (henri-tremblay) [inline voltron-proxy/voltron-proxy-client/src/main/java/org/terracotta/voltron/proxy/client/VoltronProxyInvocationHandler.java]: Diamond operator?

**Comment by henri-tremblay** [comment]
All good for me appart from one tiny comment

File: voltron-proxy/voltron-proxy-client/src/main/java/org/terracotta/voltron/proxy/client/VoltronProxyInvocationHandler.java (hunk 1 above)
Comment (chrisdennis) [inline voltron-proxy/voltron-proxy-client/src/main/java/org/terracotta/voltron/proxy/client/VoltronProxyInvocationHandler.java]: Isn't this 1.6 code?

Comment (henri-tremblay) [inline voltron-proxy/voltron-proxy-client/src/main/java/org/terracotta/voltron/proxy/client/VoltronProxyInvocationHandler.java]: Oh! Proxy client. Probably. Forget me.

Comment (mathieucarbou) [inline voltron-proxy/voltron-proxy-client/src/main/java/org/terracotta/voltron/proxy/client/VoltronProxyInvocationHandler.java]: yes @chrisdennis this is 1.6 code (client-side)

**Pull request merged**